hbase-plugins/pom.xml

```diff
@@ -71,6 +71,12 @@
       <artifactId>hydrator-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <!-- needed by TableOutputFormat, but TableOutputFormat is provided scope -->
+    <dependency>
+      <groupId>org.apache.zookeeper</groupId>
+      <artifactId>zookeeper</artifactId>
+      <scope>compile</scope>
```

Comment:
no it has to be compile, zookeeper is not exposed by our api.  The poms should be restructured, normally this would be pulled in by the hbase dependency but the parent pom has zookeeper at provided